tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-dashboard.ts

```diff
@@ -166,6 +167,8 @@ class TfScalarDashboard extends LegacyElementMixin(ArrayUpdateHelper) {
                   tag="[[item.tag]]"
                   tooltip-sorting-method="[[_tooltipSortingMethod]]"
                   x-type="[[_xType]]"
+                  batch-size="[[featureFlags.scalarsBatchSize]]"
```

Comment:
How does this behave when the `featureFlags` is undefined? (since it can be undefined as per L213)

Reply:
batchSize and inColab end up being undefined and the scalars dashboard behaves as though they aren't set. That wasn't explicitly my intention but seems like the correct behavior? Thanks for asking me to check.

Reply:
Got it. I am just surprised that `undefined.scalarsBatchSize` is legal in Polymerism but I forget Polymer :)

Reply:
Yes, I am also (pleasantly) surprised.